### ISSUE TITLE
Does reproduciblity works for GPU

### ISSUE THREAD

**Issue by cometta** [ray-project/ray#9570]
ray 0.8.6, python 3.7, pytorch 1.5.1, linux

I tested setting seed with smaller timesteps_total <20000, it works, but when i have longer timesteps_total >100000. It does not work on GPU. Anyone able to get it working on tensorflow/pytorch? Can share the config parameters
